### ISSUE TITLE
Should the width for the content be increased?

### ISSUE THREAD

**Issue by MaikKlein** [rust-lang/crates.io#1559]
First of all I am not a webdev.

I always felt that crates.io looks a bit cramped on my desktop machine.

For example this is how it currently looks

![](https://i.imgur.com/oeF0coM.png)

After I changed the width

![](https://i.imgur.com/lqyNWx0.png)

Now it has the same size as Github. I personally think this would be an improvement, what do you think?

**Comment by carols10cents** [comment]
Having the README be the same width as GitHub, at least for crate show pages, would probably be nice. Not sure if that should extend to search pages or settings pages or the other pages we have, and if not, if it would be weird that the widths change...

**Comment by moonheart08** [comment]
Maybe make it.. not be fixed width? Please? Screen sizes above 16:9 really suffer, and a lot of screen space is wasted. Pictured is a 21:9 display viewing crates.io.
![image](https://user-images.githubusercontent.com/7806367/73993186-11354280-4917-11ea-9a12-38bd7d7bd817.png)

**Comment by miDeb** [comment]
There was already an issue for that (https://github.com/rust-lang/crates.io/issues/970), but if you go to the linked page https://crates.io/crates/colorparse today, it looks more like the "before" in the gif of the closing pr https://github.com/rust-lang/crates.io/pull/1064. Maybe that's even a regression?

**Comment by ibaryshnikov** [comment]
Hi all! What's the state of this issue? Any desire to make a change for wider screens? It looks like here's the place responsible for the docs width
https://github.com/rust-lang/crates.io/blob/575b2fb97a492624e75e550fdebae209d1904906/app/styles/crate/version.module.css#L13-L19

**Comment by Turbo87** [comment]
we're slowly working towards a state of the code where we should be able to increase the width without too many issues :)

**Comment by ibaryshnikov** [comment]
Does it break something else when increased?

**Comment by Turbo87** [comment]
@ibaryshnikov yes, see https://github.com/rust-lang/crates.io/issues/3687#issuecomment-857418731 for example 😅

**Comment by hamidrezakp** [comment]
What if we support multiple fixed content widths? like Wikipedia, an standard and a wide option.

standard can be the current size but the wide one would fill the entire page.

This would be great in the `code` tab when we want to read codes and have files open next to it.
